src/pages/contact/index.astro

```diff
@@ -18,24 +18,24 @@ import Main from '../../layouts/MainContact.astro'
         </style>
         <g id="VivicVenn" transform="translate(-595.282 -254.216)">
 
-            <a id="PinkCircleClick" href="inez">
+            <a id="PinkCircleClick" href="/contact/inez">
```

Comment:
I'm going to come back to the contact page in Revision 2, but I noticed that the links 404-ed so I've fixed the `href`s here.